test/unit/view.js

```diff
@@ -5,67 +5,105 @@ import Iframe from '../../src/iframe';
 
 describe('View class', () => {
   describe('constructor', () => {
-    const component = new Component({id: 1234});
-    const view = new View(component);
+    let component;
+    let view;
+
+    beforeEach(() => {
+      component = new Component({
+        id: 1234,
+        node: document.createElement('div'),
+      });
+      view = new View(component);
+    });
 
     it('stores component to instance', () => {
       assert.equal(view.component, component);
     });
 
+    it('sets iframe to null', () => {
+      assert.equal(view.iframe, null);
+    });
+
+    it('sets node to component\'s node', () => {
+      assert.equal(view.node, component.node);
+    });
+
     it('creates a template instance', () => {
       assert.instanceOf(view.template, Template);
     });
+
+    it('sets eventBound to false', () => {
+      assert.equal(view.eventsBound, false);
+    });
   });
 
   describe('prototype methods', () => {
+    let component;
+    let view;
+
+    beforeEach(() => {
+      Component.prototype.typeKey = 'product';
```

Comment:
Since we aren't re-instantiating the component in the subsequent tests, let's set the `typeKey` in the instance of the component instead of modifying the prototype.